src/main/java/org/privacyidea/PrivacyIDEA.java

```diff
@@ -356,6 +362,48 @@ private void appendRealm(Map<String, String> params)
         }
     }
 
+    /**
+     * Retrieve the auth token from the /auth endpoint and schedule the next retrieval.
+     */
+    private void retrieveAuthToken()
+    {
+        String response = runRequestAsync(ENDPOINT_AUTH, serviceAccountParam(), Collections.emptyMap(), false, POST);
```

Comment:
latch needs to be set again for this to work more than once?

Reply:
[9f77a94](https://github.com/privacyidea/java-client/pull/70/commits/9f77a940dbe44747fdbea5e9b032a0a3782d15fb)